src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/TagHelperSemanticRangeVisitor.cs

```diff
@@ -44,16 +44,26 @@ private void Visit(SyntaxList<RazorSyntaxNode> syntaxNodes)
         }
 
         #region HTML
+        public override void VisitMarkupTextLiteral(MarkupTextLiteralSyntax node)
+        {
+            AddSemanticRange(node, RazorSemanticTokensLegend.MarkupTextLiteral);
+        }
+
+        public override void VisitMarkupLiteralAttributeValue(MarkupLiteralAttributeValueSyntax node)
+        {
+            AddSemanticRange(node, RazorSemanticTokensLegend.MarkupAttributeValue);
+        }
+
         public override void VisitMarkupAttributeBlock(MarkupAttributeBlockSyntax node)
         {
             Visit(node.NamePrefix);
             AddSemanticRange(node.Name, RazorSemanticTokensLegend.MarkupAttribute);
             Visit(node.NameSuffix);
             AddSemanticRange(node.EqualsToken, RazorSemanticTokensLegend.MarkupOperator);
 
-            Visit(node.ValuePrefix);
+            AddSemanticRange(node.ValuePrefix, RazorSemanticTokensLegend.MarkupOperator);
```

Comment:
Fun fact: as far as I can tell literally nothing was colorizing these attribute quotes before, not even TextMate.

Reply:
oh interesting